pkg/apis/vpc/v1alpha1/subnet_types.go

```diff
@@ -104,6 +108,23 @@ type SubnetDHCPConfig struct {
 	Mode DHCPConfigMode `json:"mode,omitempty"`
 }
 
+// AdvancedConfig is Subnet advanced configuration.
+type AdvancedConfig struct {
+	// StaticIPAllocation configuration for subnet ports with VIF attachment.
```

Comment:
Subnet Ports?

Maybe change VIF attachment to "Network Interface attachment" as we do not expose VIF in net-op APIs.